### PR TITLE
Translate 1 file to ko, playground - Nominal Typing

### PR DESCRIPTION
**Pull request by yahma25**
en
* [Nominal Typing.ts](https://github.com/microsoft/TypeScript-Website/blob/v2/packages//playground-examples/copy/en/TypeScript/Language%20Extensions/Nominal%20Typing.ts)

번역 리뷰 요청합니다: @yeonjuan @Bumkeyy 
ref #6

감사합니다 😄

### CODE REVIEW

File: docs/playground/ko/TypeScript/Language Extensions/Nominal Typing.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+// 함수를 사용하지만,
+// 주목할 점은 TypeScript에게 참이라고 _알려주는 것_ 입니다.
+
+const validateUserInput = (input: string) => {
+  const simpleValidatedInput = input.replace(/\</g, "≤");
+  return simpleValidatedInput as ValidatedInputString;
+};
+
+// 이제 새로운 명목적 타입이고 일반 문자열 타입이 아닌 것만
+// 허용하는 함수를 만들 수 있습니다.
```
Comment (yeonjuan) [inline docs/playground/ko/TypeScript/Language Extensions/Nominal Typing.ts:40]:
```suggestion
// 이제 일반 문자열 타입이 아닌 새로운 명목적 타입만
// 허용하는 함수를 만들 수 있습니다.
```

**Review by yeonjuan — APPROVED** [review]
고생하셨습니다. 다소 어색한 부분 제안 남겼습니다..

**Comment by yahma25** [comment]
@yeonjuan 
리뷰 수고하셨습니다 👍 
제안 반영하였습니다

**Comment by yeonjuan** [comment]
LGTM

**Comment by orta** [comment]
So strange, these two are running the same code! https://github.com/microsoft/TypeScript-DOM-lib-generator/pull/1100#issuecomment-896782590

**Pull request merged**